src/index.js

```diff
@@ -65,13 +70,58 @@ export default class TagInput extends React.Component {
     }
   }
 
+  fetchServer(query) {
+    const { fetchUrl, server } = this.props;
+
+    this.setState({
+      waiting: true
+    });
+
+    return new Promise((resolve, reject) => {
+      superagent
+        .get(fetchUrl)
+        .set('x-jira-server', server)
+        .accept('application/json')
+        .query({ query: query })
+        .end((err, res) => {
+          if (err) {
+            reject(err);
+          }
+          resolve(res.body);
+        });
+    });
+  }
+
+  handleGetOptions(query = '', active = false) {
+    if (active || this.state.data.some(field => field.label === query)) {
+      return;
+    }
+
+    this.fetchServer(query)
+      .then((res) => {
+        this.setState({
+          data: res.suggestions,
```

Comment:
We're assuming there's always going to be a `suggestions` key here. We should add another prop accepts a function that lets us extract the data we want.